tests/dragonfly/utility.py

```diff
@@ -189,8 +189,8 @@ def gen_shrink_cmd(self):
         ('SETRANGE {k} 10 {val}', ValueType.STRING),
         ('LPUSH {k} {val}', ValueType.LIST),
         ('LPOP {k}', ValueType.LIST),
-        # ('SADD {k} {val}', ValueType.SET),
-        # ('SPOP {k}', ValueType.SET),
+        ('SADD {k} {val}', ValueType.SET),
+        ('SPOP {k}', ValueType.SET),
```

Comment:
great is it also simple as that to add the expire command ?

Reply:
For commands that do not add or delete keys - yes.

Not for `EXPIRE`, because it changes the key set. It must be added as a shrink action